docs/template/data/demos.json

```diff
@@ -1,10 +1,11 @@
 [
     {"filename": "local.html", "title": "Local Data"},
+    {"filename": "pager.html", "title": "Pagination"},
+    {"filename": "remote.html", "title": "Remote"},
```

Comment:
BTW: The remote demo is still not working, see: https://github.com/openpsa/jsgrid/issues/46#issuecomment-71086732